### PR TITLE
api: Rename Title.Times.of

### LINKED ISSUE #448: Rename `Title.Times.of`
In order to be consistent with the rest of the API, this method should be named `times`.

### PR DESCRIPTION
**Pull request by NoahvdAa**
Fixes #448.

### CODE REVIEW

File: api/src/main/java/net/kyori/adventure/title/Title.java [hunk 1]
```diff
@@ -120,8 +120,23 @@
      * @param fadeOut the fade-out time
      * @return times
      * @since 4.0.0
+     * @deprecated for removal since 4.10.0, use {@link #times()}
      */
+    @Deprecated
```
Comment (kezz) [inline api/src/main/java/net/kyori/adventure/title/Title.java:125]: Should have an `@ApiStatus.ScheduledForRemoval` annotation here.

**Review by kezz — CHANGES_REQUESTED** [review]
Thanks for the PR! This PR should however also replace all usages of the now deprecated method in the codebase.

File: api/src/main/java/net/kyori/adventure/title/Title.java [hunk 2]
```diff
@@ -120,8 +120,24 @@
      * @param fadeOut the fade-out time
      * @return times
      * @since 4.0.0
+     * @deprecated for removal since 4.10.0, use {@link #times()}
      */
+    @ApiStatus.ScheduledForRemoval
```
Comment (Proximyst) [inline api/src/main/java/net/kyori/adventure/title/Title.java:125]: Perhaps add an `inVersion` attribute?

Comment (kezz) [reply api/src/main/java/net/kyori/adventure/title/Title.java:125]: Ooh yes I didn't even know this was a thing! Definitely add one of these with `5.0.0` as the version!

**Review by kezz — APPROVED** [review]
Thanks for the PR :)

**Review by zml2008 — APPROVED** [review]

**Pull request merged**